verification/cbmc/proofs/aws_hash_callback_c_str_eq/aws_hash_callback_c_str_eq_harness.c

```diff
@@ -12,6 +12,9 @@
 void aws_hash_callback_c_str_eq_harness() {
     const char *str1 = ensure_c_str_is_allocated(MAX_STRING_LEN);
     const char *str2 = nondet_bool() ? str1 : ensure_c_str_is_allocated(MAX_STRING_LEN);
+
+    __CPROVER_assume(aws_c_string_is_valid(str1) && aws_c_string_is_valid(str2));
```

Comment:
Is there a reason you conjoin these rather than having two assumes?

Reply:
No reason, I split into two assumes.